locale/nl.po

```diff
@@ -3425,9 +3413,9 @@ msgid "Failed to load shared library '%s'"
 msgstr "Laden van gedeelde bibliotheek '%s' mislukt"
 
 #: ../src/osx/core/sound.cpp:118
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to load sound from \"%s\" (error %d)."
-msgstr "Laden van bronbestand \"%s\" mislukt."
+msgstr "Laden geluid van \"%s\" mislukt (fout %d)."
 
```

Comment:
Personally I'd prefer "Laden van geluid \"%s\" [...]" or "Laden geluid vanaf \"%s\" [...]" . It sounds a little less 'krom' IMHO.